File: 3289-the-two-sneaky-numbers-of-digitville/3289-the-two-sneaky-numbers-of-digitville.cpp
class Solution {
public:
    vector<int> getSneakyNumbers(vector<int>& nums) {
        int n = nums.size();
        vector<int>freq(n - 2, 0);
        vector<int>ans;
        for (const int& i: nums) {
            freq[i]++;
            if (freq[i] >= 2) {
                ans.push_back(i);
            }
        }
        return ans;
    }
};

File: 3289-the-two-sneaky-numbers-of-digitville/3289-the-two-sneaky-numbers-of-digitville.py
class Solution:
    def getSneakyNumbers(self, nums: List[int]) -> List[int]:
        n = len(nums)
        freq = [0] * (n - 2)
        ans = []
        for num in nums:
            freq[num]+=1
            if freq[num] >= 2:
                ans.append(num)
        return ans